doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md

```diff
@@ -5,4 +5,102 @@ permalink: /:path/
 sort_idx: 20
 ---
 
-SDK integration instructions for select platforms
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+# The guide provides information on how to cross-compile C++ SDK for Raspberry PI
+
+**Verified against:**
+
+**Host OS:** Ubuntu 14.04 LTS Desktop 64-bit.
+
+**The further instructions must be executed on the host machine.**
+
+1. Download and install toolchain.
+
+        mkdir rpi_root && cd rpi_root
+        export RPI_ROOT=$(pwd)
+        git clone https://github.com/raspberrypi/tools.git
+        export INSTALLDIR=$RPI_ROOT/tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian
```

Comment:
It may be worth setting INSTALLDIR to `$RPI_ROOT/tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian/arm-linux-gnueabihf/libc/usr` given how it used below.